thewalrus/tests/test_samples.py

```diff
@@ -429,6 +453,7 @@ def test_thermal_state_torontonian(self, sample_func):
     @pytest.mark.parametrize("parallel", [True, False])
     def test_torontonian_sample_graph(self, parallel):
         """Test torontonian sampling from a graph"""
+        np.random.seed(42)
```

Comment:
How come this is needed here? I noticed that there's also a `seed` function imported on line 28, that basically does the exact same thing. Just for consistency maybe that could be used here as well.

Reply:
We put it in whilst debugging some things. Tests were passing on my PC but not in the CI. I'll try to see if I can get the CI tests to pass without the seed.

Reply:
okay, tests seem to pass without the seed! :D